Mixpanel/InAppNotifications.swift

```diff
@@ -79,20 +82,32 @@ class InAppNotifications: NotificationViewControllerDelegate {
     }
 
     @discardableResult
-    func notificationShouldDismiss(controller: BaseNotificationViewController, callToActionURL: URL?) -> Bool {
+    func notificationShouldDismiss(controller: BaseNotificationViewController,
+                                   callToActionURL: URL?,
+                                   shouldTrack: Bool,
+                                   additionalTrackingProperties: Properties?) -> Bool {
         if currentlyShowingNotification?.ID != controller.notification.ID {
             return false
         }
 
         let completionBlock = {
+            if shouldTrack {
+                var properties = additionalTrackingProperties
+                if let urlString = callToActionURL?.absoluteString {
+                    if properties == nil {
```

Comment:
is there a reason why we don't leave this nil and send it like that? there is a unwrapping in trackNotifcation

Reply:
i wanted it to always send "url"